tests/regression/roles/sonic_interfaces/defaults/main.yml

```diff
@@ -41,18 +42,22 @@ tests:
         description: ansible Ethernet96
         mtu: 6444
         enabled: true
+        unreliable_los: UNRELIABLE_LOS_MODE_AUTO
```

Comment:
Change this to one of the other values to verify that changing the value works correctly.

Reply:
I have changed the value.